core/test/market-price-test.ts

```diff
@@ -14,17 +13,7 @@ describe('Market Price & Conversions', () => {
         await setupRpcUrlAndGetNetworks(local_rpc_url);
     });
     beforeEach(async () => {
-        await hre.network.provider.request({
-            method: 'hardhat_reset',
-            params: [
-                {
-                    forking: {
-                        jsonRpcUrl: REMOTE_RPC_URL,
-                        blockNumber: HARDHAT_FORK_BLOCK_NUMBER,
-                    },
-                },
-            ],
-        });
+        await resetNetwork(REMOTE_RPC_URL, HARDHAT_FORK_BLOCK_NUMBER);
```

Comment:
Why can't you use `default` export from `resetBlockchainFork` here instead of hardcoding `http://localhost:8545` for the third time?

Reply:
irrelevant due to changes inflicted by the previous review comments i guess.